be/src/olap/tablet_manager.cpp

```diff
@@ -1091,7 +1131,48 @@ void TabletManager::try_delete_unused_tablet_path(DataDir* data_dir, TTabletId t
     // TODO(ygl): may do other checks in the future
     if (Env::Default()->path_exists(schema_hash_path).ok()) {
         LOG(INFO) << "start to move tablet to trash. tablet_path = " << schema_hash_path;
-        OLAPStatus rm_st = move_to_trash(schema_hash_path, schema_hash_path);
+        FilePathDesc segment_desc(schema_hash_path);
```

Comment:
Tablet GC could not be performance on any BE, because it does not know when to delete data.
It should be performed on FE. Be only care about its local data not remote data.

Reply:
Every remote data has a local meta, When the local meta is deleted, remote data will be deleted too.
If deleted by fe, when a migration operation is failed and did not notify fe, how can fe know it？